pallets/runtime/tests/src/storage.rs

```diff
@@ -737,8 +737,19 @@ pub fn register_keyring_account_without_cdd(
     make_account_without_cdd(acc.to_account_id()).map(|(_, id)| id)
 }
 
+pub fn get_primary_key(target: IdentityId) -> AccountId {
+    Identity::get_primary_key(target)
+}
+
+pub fn get_secondary_keys(target: IdentityId) -> Vec<SecondaryKey<AccountId>> {
```

Comment:
```suggestion
pub fn primary_key(target: IdentityId) -> AccountId {
    Identity::get_primary_key(target)
}

pub fn secondary_keys(target: IdentityId) -> Vec<SecondaryKey<AccountId>> {
```

Reply:
No.

Reply:
Why? The Rust naming conventions avoids the use of the `get_` prefix, https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
Your reason is invalid because that policy applies to methods.  These are stand-alone functions.